src/main/java/com/intel/genomicsdb/GenomicsDBImporter.java

```diff
@@ -23,1615 +23,1665 @@
+     * @param workspace              TileDB workspace
+     * @param arrayname              TileDB array name
+     * @param sizePerColumnPartition sizePerColumnPartition in bytes
+     * @param segmentSize            segmentSize in bytes
+     * @throws IOException FeatureReader.query can throw an IOException if invalid file is used
+     */
+    public GenomicsDBImporter(Map<String, FeatureReader<VariantContext>> sampleToVCMap,
+                              Set<VCFHeaderLine> mergedHeader,
+                              ChromosomeInterval chromosomeInterval,
+                              String workspace,
+                              String arrayname,
+                              Long sizePerColumnPartition,
+                              Long segmentSize) throws IOException {
+        this(sampleToVCMap, mergedHeader, chromosomeInterval,
+                workspace, arrayname, sizePerColumnPartition, segmentSize,
+                (long) 0, Long.MAX_VALUE - 1, true);
+    }
+    /**
+     * Constructor with an GenomicsDB import configuration protocol buffer
+     * structure. Avoids passing long list of parameters. This constructor
+     * is developed specifically for GATK4 GenomicsDBImport tool.
+     *
+     * @param sampleToVCMap             Variant Readers objects of the input GVCF files
+     * @param mergedHeader              Set of VCFHeaderLine from the merged header across all input files
+     * @param chromosomeInterval        Chromosome interval to traverse input VCFs
+     * @param importConfiguration       Protobuf configuration object containing related input
+     *                                  parameters, filenames, etc.
+     * @param validateSampleToReaderMap Check validity of sampleToreaderMap entries
+     * @throws IOException Throws file IO exception.
+     */
+    public GenomicsDBImporter(Map<String, FeatureReader<VariantContext>> sampleToVCMap,
+                              Set<VCFHeaderLine> mergedHeader,
+                              ChromosomeInterval chromosomeInterval,
+                              boolean validateSampleToReaderMap,
+                              GenomicsDBImportConfiguration.ImportConfiguration importConfiguration)
+            throws IOException {
+        this(sampleToVCMap,
+                mergedHeader,
+                chromosomeInterval,
+                importConfiguration.getColumnPartitions(0).getWorkspace(),
+                importConfiguration.getColumnPartitions(0).getArray(),
+                importConfiguration.getSizePerColumnPartition(),
+                importConfiguration.getSegmentSize(),
+                importConfiguration.getGatk4IntegrationParameters().getLowerSampleIndex(),
+                importConfiguration.getGatk4IntegrationParameters().getUpperSampleIndex(),
+                false,  //useSamplesInOrderProvided
+                importConfiguration.getFailIfUpdating(),
+                validateSampleToReaderMap
```

Comment:
Did any of the constructors change? As mentioned above, the big differences in file layouts makes review difficult. Hard to check if any arguments etc changed here.

Reply:
Nop.